pages/about.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;900&family=Poppins:wght@100&display=swap"
+        rel="stylesheet">
+    <title>thejunglemanics</title>
+</head>
+
+<body>
+    <section>
+        <div class="top-container">
+            <div class="header-container">
+                <header class="desktop-topbar">
+                    <div>
+                        <a><i class="fa-brands fa-facebook-f"></i></a>
+                        <a><i class="fa-brands fa-twitter"></i></a>
+                        <a>English</a>
+                        <a>Sign Up</a>
+                        <a>Login</a>
+                    </div>
+                </header>
+                <nav class="">
+                    <div class="desktop-menu">
+                        <div class="header-logo"><a href='/index.html' class="header-logo"><img
+                                    src="/JM-Images/JunglemanicLogo.png"></a></div>
+                        <div><a class="about-h4" href="#top-section">About</a></div>
+                        <div>
+                            <h4><a href="#activities">Main Activities</a></h4>
```

Comment:
Currently, your about page does not represent the behance design at all. Please update your about page to include all the elements present in the behance design. Please make sure to respect the color scheme, typography and layout of each element

Your design:
<img width="435" alt="image" src="https://github.com/pushkar1995/capstone-project1.0/assets/37116322/142c1ec0-79ba-4219-9d5b-9385e5ee36a6">

Behance design:
<img width="527" alt="image" src="https://github.com/pushkar1995/capstone-project1.0/assets/37116322/aaaad476-f35c-4421-8ea9-a2a86883d0f3">